### PR TITLE
[Merged by Bors] - Small README fix

### PR DESCRIPTION
**Pull request by vsupalov**
# Description

The README contained a broken sentence. This PR fixed it.

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,7 @@ If you have a question about the Stackable Data Platform contact us via our [hom
 
 ## What Does This Do?
 
-This operator is managing Kafka in Kubernetes. It is part of the Stackable Data Platform, a 
-
-Operators observe custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
+This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
```
Comment (lfrancke) [inline README.md:39]:
```suggestion
This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manage the application it's responsible for according to the desired state specified in those custom resources.
```

File: .readme/partials/main.md.j2 [hunk 2]
```diff
@@ -16,9 +16,7 @@ You can follow this [tutorial](https://docs.stackable.tech/kafka/stable/getting_
 
 ## What Does This Do?
 
-This operator is managing Kafka in Kubernetes. It is part of the Stackable Data Platform, a 
-
-Operators observe custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
+This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
```
Comment (lfrancke) [inline .readme/partials/main.md.j2:19]:
```suggestion
This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manage the application it's responsible for according to the desired state specified in those custom resources.
```

**Review by lfrancke — CHANGES_REQUESTED** [review]

File: .readme/partials/main.md.j2 [hunk 3]
```diff
@@ -16,7 +16,7 @@ You can follow this [tutorial](https://docs.stackable.tech/kafka/stable/getting_
 
 ## What Does This Do?
 
-This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
+This operator is managing Kafka in Kubernetes. Operators observe custom resources in a Kubernetes cluster, and manage the application it's responsible for according to the desired state specified in those custom resources.
```
Comment (lfrancke) [inline .readme/partials/main.md.j2:19]: This still reads wrong....

Maybe like this?
```suggestion
This operator is managing Kafka in Kubernetes. An operator observes custom resources in a Kubernetes cluster, and manages the application it's responsible for according to the desired state specified in those custom resources.
```

Comment (lfrancke) [reply .readme/partials/main.md.j2:19]: This is just a suggestion but my earlier proposal still lead to bad english.

Comment (vsupalov) [reply .readme/partials/main.md.j2]: Both read about right to me. Let's go with the proposed change.

**Review by lfrancke — APPROVED** [review]
Thank you!

**Pull request closed without merging**